src/components/login/MagicLink.tsx

```diff
@@ -68,6 +69,10 @@ const MagicLink = () => {
 
     custom_generateDefaultUISchema;
 
+    const redirectTo = `${redirectToBase}?${REDIRECT_TO_PARAM_NAME}=${encodeURIComponent(
+        window.location.pathname
```

Comment:
Given we have some paths that use search params we should probably include search here.

Example: We put the selected connector id in the URL during create.

Reply:
Yuup, good point. Done!